CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.1.3
+
+### Fixes
+
+Tutorials are now numbered incorrectly in the Carbon Demo sidebar.
```

Comment:
Change the heading from `Fixes` to `Demo Site`.

And there's a typo: shouldn't that be 'correctly'?

Reply:
Poop ! Yip. Done. 👍